README.md

```diff
@@ -1 +1,49 @@
+# frontend-coding-standards
+
+## Recommended Libraries
+
+#### Animation & UI
+ * [classnames](https://www.npmjs.com/package/classnames) - A simple JavaScript utility for conditionally joining classNames together.
+ * [Draggable](https://greensock.com/draggable/) - Green Sock library for dragging elements
+ * [grid-checker](https://www.npmjs.com/package/grid-checker) - provides developers a customizable, responsive grid-overlay to compare with the layout provided by design to compare layouts
+ * [GSAP](https://greensock.com/gsap/) - Green Sock Animation Platform -Animation library
+ * [Litepicker](https://github.com/ThijsTyZ/Litepicker) - Date range picker - lightweight, no dependencies
+ * [Lottie](http://airbnb.io/lottie/) - parses Adobe After Effects animations exported as json with Bodymovin and renders them natively on mobile and on the web
+ * [scroll-tracker-component-manager](https://www.npmjs.com/package/scroll-tracker-component-manager) - The ScrollTrackerComponentManager is a Class that tracks whether a component is within your viewport based on your scroll position.
+ * [seng-scss](https://www.npmjs.com/package/seng-scss) - A SCSS library of mixins and functions
+ * [seng-device-state-tracker](https://github.com/mediamonks/seng-device-state-tracker) - DeviceStateTracker is a utility class that tracks which media query is currently active using the matchMedia API.
+ * [storybook](https://github.com/storybookjs/storybook) - UI component dev & test
+
+#### Coding
+ * [bowser](https://www.npmjs.com/package/bowser) - A small, fast and rich-API browser/platform/engine detector
+ * [date-fns](https://www.npmjs.com/package/date-fns) - provides the most comprehensive, yet simple and consistent toolset for manipulating JavaScript dates
+ * [i18next](https://www.npmjs.com/package/i18next) - Internationalization library
+ * [lodash](https://lodash.com/) - A modern JavaScript utility library delivering modularity, performance & extras.
+ * [Moment](https://www.npmjs.com/package/moment) - A lightweight JavaScript date library for parsing, validating, manipulating, and formatting dates.
+ * [seng-disposable-event-listener](https://www.npmjs.com/package/seng-disposable-event-listener) - event listening management
+ * [seng-disposable-manager](https://www.npmjs.com/package/seng-disposable-manager) - event management
+ * [seng-event](https://www.npmjs.com/package/seng-event) - Provides Classes and utilities for dispatching and listening to events.
+ * [Yup](https://github.com/jquense/yup) - Form validation
+
+#### TypeScript
+ * [ts-essentials](https://www.npmjs.com/package/ts-essentials) - TypeScript utilities
+
+#### React
+ * [Formik](https://jaredpalmer.com/formik/) - Forms
+ * [MobX](https://mobx.js.org/README.html) - Simple, scalable state management
+ * [React Router](https://reacttraining.com/react-router/) - Router library for React
+ * [Redux](https://redux.js.org/) - A Predictable State Container for JS Apps
```

Comment:
- redux-thunk
- typesafe-actions

Reply:
I think redux would need its own section of sub-libraries :)

Reply:
I would not ever recommend moment because of mutability and importing the whole moment object. Date-fns solves date handling in a much cleaner way. The codebase then deals with native Date objects, not Moment objects and you only import the functions you need. Manipulation is very clear because you always get a new Date. With moment if you need to remember to clone the object...